### DISCUSSION TITLE
[@types/aws4] version 1.11.0 produce an error during build

### DISCUSSION THREAD

**Discussion by DoodahProductions** [DefinitelyTyped/DefinitelyTyped#56628] [Issues with a @types package]
Hello,

Please note that version 1.11.0 produce the following error during builds

`
node_modules/@types/aws4/index.d.ts:95:38 - error TS1005: ',' expected.
95     matchHost(host: string): [service: string, region: string];
                                        ~

node_modules/@types/aws4/index.d.ts:95:54 - error TS1005: ',' expected.
95     matchHost(host: string): [service: string, region: string];         
`

[a comment by an automated account is not shown]

**Comment by qexk** [comment]
Hello, sorry for the late answer
The latest version uses labeled tuples for this return type, so the minimal TypeScript version has been bumped to 4.0

**Comment by qexk** [comment]
Hello @DoodahProductions, a PR was merged yesterday (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57966) to remove that named tuple.
Does it fix the issue you were having?
